lib/ukrust/ukrust_sys/lib.rs

```diff
@@ -49,3 +49,15 @@ pub mod c_types;
 pub fn alloc_error(_layout: core::alloc::Layout) -> ! {
         panic!("Alloc error");
 }
+
+extern "C" {
+    fn __ukrust_sys_crash() -> !;
+}
+
+#[panic_handler]
+fn panic(_info: &core::panic::PanicInfo<'_>) -> ! {
+    unsafe {
+        __ukrust_sys_crash();
+    }
+}
```

Comment:
line 54: ```warning: `__ukrust_sys_crash` redeclared with a different signature``` 
It is saying about `__ukrust_sys_crash` exported by `bindings`.

Can we remove the explicit ` -> !` to make the warning go away and insert a `loop {}` in the panic handler?

Like the examples from https://doc.rust-lang.org/nomicon/panic-handler.html

Reply:
Hi Fabian, I would keep it like this right now, since this would be a fix, and this pull request is related only to moving. Maybe add it in a separate pull request, but I remember they didn't use the `loop {}` in Linux neither, so I wonder why.

Reply:
Hi Vlad,

I see Linux using `loop {}` this way inside the `panic_handler`. [1]
This is, however, a workaround for a reason mentioned there ([1]) in the comments.

Let's add it next time then.

[1] - https://github.com/torvalds/linux/blob/041fae9c105ae342a4245cf1e0dc56a23fbb9d3c/rust/kernel/lib.rs#L78-L87

Reply:
Cool, they updated it! Yeah, we should do an upgrade in the future.